tests/lib/errors/CombinedPropertyError.test.ts

```diff
@@ -0,0 +1,61 @@
+import { inspect } from 'node:util';
+import { CombinedPropertyError, ValidationError } from '../../../src';
+
+describe('CombinedError', () => {
+	const error = new CombinedPropertyError([
+		['foo', new ValidationError('StringValidator', 'Expected a string primitive', 42)],
+		[2, new ValidationError('StringValidator', 'Expected a string primitive', true)],
+		[Symbol('hello.there'), new ValidationError('StringValidator', 'Expected a string primitive', 75n)]
+	]);
+
+	test('GIVEN an instance THEN assigns fields correctly', () => {
+		expect(error.message).toBe('Received one or more errors');
+		expect(error.errors).toHaveLength(3);
+		expect(error.errors[0][1]).toBeInstanceOf(ValidationError);
+		expect(error.errors[1][1]).toBeInstanceOf(ValidationError);
+		expect(error.errors[2][1]).toBeInstanceOf(ValidationError);
+	});
+
+	describe('inspect', () => {
+		test('GIVEN an inspected instance THEN formats data correctly', () => {
+			const content = inspect(error, { colors: false });
+			const expected = [
+				'CombinedPropertyError (3)',
+				'  Received one or more errors',
+				'',
+				'  .foo',
+				'  | ValidationError > StringValidator',
+				'  |   Expected a string primitive',
+				'  | ',
+				'  |   Received:',
+				'  |   | 42',
+				'',
+				'  [2]',
```

Comment:
[2] feels weird.. is it input[2]? Is it just the value 2?

Reply:
`input[2]`